en/lightpaper/FEATURE.tex

```diff
@@ -94,9 +94,9 @@ \section{GitHub, Open-Source, and FEATURE}
 
 \section{Atomic Contract}
 
-The idea of making payments in cryptocurrency originated in 2008 with the publication of the "Bitcoin" whitepaper. Cryptocurrency allows for peer-to-peer transactions in a secure manner, without the need for a trusted third party. This means that users can make payments directly to each other, without relying on an intermediary to validate or verify transactions.
+The idea of making payments in cryptocurrency originated in 2008 with the publication of the ``Bitcoin'' whitepaper. Cryptocurrency allows for peer-to-peer transactions in a secure manner, without the need for a trusted third party. This means that users can make payments directly to each other, without relying on an intermediary to validate or verify transactions.
 
-This concept was extended to the creation of smart contracts, allowing users or programs to interact with programs that "automatically" execute predefined actions when specific conditions are met.
+This concept was extended to the creation of smart contracts, allowing users or programs to interact with programs that ``automatically'' execute predefined actions when specific conditions are met.
```

Comment:
Not sure about this

Reply:
This is the standard way to write quotation marks and double quotes in LaTeX.

The \quotationmacro is language specific, so it gives you the correct quotation marks in English ( “—”), français ( « — »)
```txt
\quotation{...}
```